CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- 🎁 Analyzer plugins can now communicate with the node directly.
+  [#1338](https://github.com/tenzir/vast/pull/1338)
```

Comment:
Not a user-facing change; we currently don't have a developer changelog.

Reply:
I removed the commit that added the changelog entry.